oem/firstvoices/keyboards.csv

```diff
@@ -7,16 +7,16 @@ fv,fv_uummarmiutun,Uummarmiutun,Arctic,fv_uummarmiutun_kmw-9.0.js,9.1.1,ikt-Latn
 fv,fv_eastern_canadian_inuktitut,ᐃᓄᒃᑎᑐᑦ (Eastern Canadian Inuktitut),Arctic,fv_eastern_canadian_inuktitut_kmw-9.0.js,9.2.1,ike-Cans,Eastern Canadian Inuktitut (Unified Canadian Aboriginal Syllabics)
 fv,fv_migmaq,Mi'gmawi'simg / Mi'kmawi'simk,Atlantic,fv_migmaq_kmw-9.0.js,9.1.2,mic-Latn,Mi'kmaq (Latin)
 fv,fv_skicinuwatuwewakon,Skicinuwatuwewakon,Atlantic,fv_skicinuwatuwewakon_kmw-9.0.js,9.1.1,pqm-Latn,Malecite-Passamaquoddy (Latin)
-fv,fv_uwikala,'Uwik̓ala,BC Coast,fv_uwikala_kmw-9.0.js,9.4,hei,Heiltsuk
+fv,fv_uwikala,’Wuìk̓ala,BC Coast,fv_uwikala_kmw-9.0.js,9.4,hei,Heiltsuk
```

Comment:
all the U+0027 : APOSTROPHE  have been replaced in this commit with single right quotes U+2019 : RIGHT SINGLE QUOTATION MARK. Is that deliberate or a text editor auto correct?

Reply:
I was trying to match the keyboard names with what's displayed on the keyboard page
https://keyman.com/keyboards/fv_uwikala

The keyboard page should be using the names as defined in the .kmn files.

And I only did a brief pass through the keyboard list, so didn't update all of them.

Reply:
Currently, the 18.0 Android app gets the keyboard names from the fv_all.kmp kmp.json file.

17.0 Android app and the iOS FV apps still use keyboard names defined in keyboards.csv